spec/features/track_members_worked_hours_spec.rb

```diff
@@ -76,4 +74,13 @@
       it 'shows that member on the admin dashboard'
     end
   end
+
+  def set_enrollments_for_hours_resume_test
+    slot = 1.week.ago.clamp(Date.current.at_beginning_of_month, Date.current)
+    3.times do
+      create :enrollment, member: member,
+                          mission: create(:mission, start_date: slot, due_date: slot + 3.hours)
+      slot -= 1.month
+    end
+  end
```

Comment:
`resume` in french means either `CV` or `reprendre`. This method name is very confusing and I think it doesn't convey its purpose. Maybe you should rename it as what it does exactly, like `set_enrollments_for_the_last_three_months` (if I understood correctly)
Since that to know what this helper does, we have to scroll down, the name either have to be very explicit or the helper should not exist in the first place (and thus this setup has to be put in a `before` block, since this setup is only used for one given `context`)

Reply:
done